### PR TITLE
doc: update quick start for full rust example

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

File: docs/src/basic.md [hunk 1]
```diff
@@ -11,43 +11,59 @@
 ## Installation
 
 === "Python"
+
```
Comment (eddyxu) [inline docs/src/basic.md:14]: Seems to make vscode happy without crazy reformatting

File: docs/src/basic.md [hunk 2]
```diff
@@ -88,6 +149,7 @@ Sometimes you may not have the data to insert into the table at creation time.
 In this case, you can create an empty table and specify the schema.
 
 === "Python"
+
```
Comment (QianZhu) [inline docs/src/basic.md:152]: nit: should we also add js and rust code

**Comment by eddyxu** [comment]
Nodejs document tests failed on main as well.

File: docs/src/basic.md (hunk 2 above)
Comment (eddyxu) [reply docs/src/basic.md:152]: Oh i thought javascript / rust can not do it yet. I can double check

Comment (QianZhu) [reply docs/src/basic.md:152]: also, should we add to our workflow to check the example rust code?

Comment (QianZhu) [reply docs/src/basic.md:152]: javascript should be able to create an empty table

Comment (eddyxu) [reply docs/src/basic.md:152]: Yea that'd be great. https://github.com/lancedb/lancedb/issues/875

**Review by QianZhu — APPROVED** [review]

**Pull request merged**